data/map planets.txt

```diff
@@ -128,14 +129,25 @@ planet Allhome
 	security 0.6
 
 planet "Alta Hai"
-	attributes "hai quarg" quarg station ringworld
+	attributes "hai quarg" quarg ringworld station
 	landscape land/station19
 	description `This is a complete Quarg ringworld, the product of millennia of construction effort which has consumed every other stellar object in this system in order to provide the raw materials. The inner edge of the ring is filled with solar panels, hydroponic gardens, living quarters, and sunlit common spaces. The outer edge contains hangars and docks for ships, and storage facilities for cargo.`
 	description `	Massive station-keeping thrusters on the ring's exterior keep its orbit from becoming unstable.`
 	spaceport `If this section of the ring is any indication, there must be trillions of Quarg here. Since the construction work on the ring has long since ended, many of them seem to spend their time with creative or scientific endeavors instead. The spaceport is decorated with artwork of every imaginable kind: sculptures, mobiles, paintings, music, and even one hallway that cycles through various artificial odors, some of them intensely unpleasant to your human senses.`
 	bribe 0
 	security 0.9
 
+planet Alvorada
+	attributes ccor military station
```

Comment:
Should these planets also have the `pirate` and `"south pirate"` attributes so that they can be sources/destinations of the normal pirate jobs, or is the expectation that these planets will have entirely unique jobs?

Reply:
Yes, they should have these tags. From the original CCOR as a unified entity concept they wouldn't but as remains the pirates would have operations in the region.

Reply:
These planets should also get the normal `south` attribute as well.

Reply:
Should the above be for both stations and planets, maybe aside from mendez?

Reply:
The normal landables should get south, pirate, and south pirate, and maybe frontier as well. Looking to other pirate planets, they all have pirate, frontier, their regional attribute, then their regional pirate attribute.

The closest thing we have to Mendez is Mutiny, the moon of Covert. Mutiny only has the pirate attribute... although it should also at least have the moon attribute as well. But given that, I'd say that Mendez only having ccor, station, and uninhabited is fine.

Reply:
What's the Frontier tag do/for?

Reply:
Adds special jobs, for example delivery jobs that tell you there is an increased pirate risk.